config/google_cloud_exporter/nginx/README.md

```diff
@@ -0,0 +1,11 @@
+# NGINX Metrics with Google Cloud
+
+The NGINX Receiver can be used to send NGINX metrics to Google Cloud Monitoring.
+
+## Limitations
+
+The collector must be installed on the NGINX system.
+
+## Prerequisites
+
+See the [prerequisites](../README.md) doc for Google Cloud prerequisites.
```

Comment:
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/nginxreceiver#configuration

We should call out that the `ngx_http_stub_status_module` should be configured in order for this receiver to work.